src/app/zap-templates/zcl/data-model/chip/global-enums.xml

```diff
@@ -61,13 +61,31 @@ TODO: Make these structures global rather than defining them for each cluster.
     <item name="Equipment" value="0x02"/>
   </enum>
 
+  <enum name="ServerAttributionContextInformation" type="enum8">
+    <item name="ServerAutomationRule" value="0x0B"/>
+    <item name="ServerSchedule" value="0x0C"/>
+    <item name="ServerTimer" value="0x0D"/>
+    <item name="Security" value="0x0E"/>
+    <item name="DemandResponseEventOrSimilarServiceProviderSignalling" value="0x0F"/>
+    <item name="PhysicalInteractionOnDevice" value="0x10"/>
+    <item name="InteractionViaAProtocolNotDefinedInThisSpecification" value="0x11"/>
+  </enum>
+
   <enum name="SoftwareVersionCertificationStatusEnum" type="enum8" apiMaturity="provisional">
     <item name="dev-test" value="0x00"/>
     <item name="provisional" value="0x01"/>
     <item name="certified" value="0x02"/>
     <item name="revoked" value="0x03"/>
   </enum>
 
+  <enum name="SuppliedAttributionContextInformation" type="enum8">
```

Comment:
Why is this being added?

Reply:
It's added unexpected by the script. I will remove it